API/DTOs/GameDTO.cs

```diff
@@ -39,6 +40,21 @@ public class GameDTO
     /// </summary>
     public long OsuId { get; set; }
 
+    /// <summary>
+    /// The current state of verification
+    /// </summary>
+    public VerificationStatus VerificationStatus { get; set; }
```

Comment:
is there a possibility to rename the system as `CurrentVerificationStatus`? so in the end it would look like `public CurrentVerificaitonStatus VerificationStatus { get; set; }`, i have a feeling this could cause some issues along the way.

The only reason I mention this is bc in my past code normally I would have to do something like this but it oculd not be the same name twice.

Reply:
That would involve renaming the enum so no. Personally I think it's fine as is considering there is no real confusion over the type of this property

Reply:
Agreed with Myssto, I don't see any issue with the enum & property name matching. Pretty common in C# afaik.

Reply:
ah im used to it not being like that in the web dev world. Outside of that everything here is fine